ci/scripts/nanoarrow_build.sh

```diff
@@ -19,6 +19,8 @@
 
 set -e
 
+# SC2034 (warning): arrow_dir appears unused.
+# shellcheck disable=SC2034
 arrow_dir=${1}
 source_dir=${1}/nanoarrow
```

Comment:
This should use `${arrow_dir}` not `${1}`.

Reply:
Use `arrow_dir` instead of `${1}`.